ga4gh/datamodel/variants.py

```diff
@@ -1096,8 +1096,11 @@ def convertVariantAnnotation(self, record, transcriptConverter):
                 annotations, variant, hgvsG, transcriptConverter)
         else:
             annotations = record.info.get('CSQ'.encode())
-            transcriptEffects = self.convertTranscriptEffectCSQ(
-                annotations[0], hgvsG)
+            transcriptEffects = []
+            for ann in annotations:
+                transcriptEffects += \
```

Comment:
Thanks @jeromekelleher the called function returns a list of transcript effects so I've used the extend method, which I hope is more clear.